style.css

```diff
@@ -461,6 +461,123 @@ span {
   margin-right: 35px;
 }
 
+.modal-container {
+  display: none;
+  flex-direction: column;
+  align-content: center;
+  width: 100%;
+  padding: 10px;
+  background-color: #979493;
+  overflow: hidden;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
```

Comment:
- [ ] Kindly make sure your modal container has a background that is identical to the one in the provided Figma design.
 
    - **YOUR DESIGN**
      ![image](https://user-images.githubusercontent.com/87197412/210881214-40d05e26-f9f0-470a-8974-7876e410ca43.png)

    - **FIGMA DESIGN**
    
      ![image](https://user-images.githubusercontent.com/87197412/210881327-f1e121d1-4cb8-4853-8bee-b142c6eab140.png)